.github/workflows/test_on_pr.yml

```diff
@@ -51,7 +63,7 @@ jobs:
         fetch-depth: 0
 
     - name: Test without coverage
-      if: matrix.platform == 'macos-latest' || matrix.platform == 'windows-latest'
+      if: matrix.platform == 'macos-latest' || matrix.platform == 'windows-latest' || matrix.platform == 'ubuntu-latest-x86'
       run: make test
 
```

Comment:
You can see that in the runner anyway, at the top...

Reply:
I don't think that would be the case for the Alpine x86 job, since it seems to run on the same worker, no?

Reply:
I don't think the new workflow would be executed until the changes haven't been merged

Reply:
I split it manually in multiple jobs anyways, so it's more clear

Reply:
Moving the changes to the CI in another PR to be merged before this one, so we have the tests running here

Reply:
Now I'm confused:
- according to the header, the `synchronize` type will launch the script every time you do a `git push`
- I can't find your changes to the file `.github/workflows/test_on_pr.yml` anymore - did you remove the changes?

I do think it's a good idea to add the 32-bit test to the workflow. And if I'm not mistaken, the workflow has actually been run. It's just that the `-x86` target doesn't exist. But I didn't test it.

Reply:
So, yes the synchronize make the CI to run for every git push, however the CI that is run is the one which is already on the master branch (or the drandmerge since it's the base branch now). Hence, I changed the workflow to execute separate jobs and the changes weren't detected either, so I moved the CI changes to a separate [PR](https://github.com/dedis/kyber/pull/523) to merge before this one so we have the correct CI tests running.

Reply:
Will update the CI with that

Reply:
Great, a good CI-pipeline is worth putting some effort in!